cmd/kubeadm/app/cmd/init.go

```diff
@@ -237,22 +244,34 @@ func AddInitConfigFlags(flagSet *flag.FlagSet, cfg *kubeadmapiv1beta1.InitConfig
 }
 
 // AddInitOtherFlags adds init flags that are not bound to a configuration file to the given flagset
-func AddInitOtherFlags(flagSet *flag.FlagSet, cfgPath *string, skipTokenPrint, dryRun *bool, ignorePreflightErrors *[]string) {
+// Note: All flags that are not bound to the cfg object should be whitelisted in cmd/kubeadm/app/apis/kubeadm/validation/validation.go
+func AddInitOtherFlags(
+	flagSet *flag.FlagSet,
+	cfgPath *string,
+	skipTokenPrint, dryRun, uploadCerts, skipCertificateKeyPrint *bool,
+	ignorePreflightErrors *[]string,
+) {
 	options.AddConfigFlag(flagSet, cfgPath)
 	flagSet.StringSliceVar(
 		ignorePreflightErrors, options.IgnorePreflightErrors, *ignorePreflightErrors,
 		"A list of checks whose errors will be shown as warnings. Example: 'IsPrivilegedUser,Swap'. Value 'all' ignores errors from all checks.",
 	)
-	// Note: All flags that are not bound to the cfg object should be whitelisted in cmd/kubeadm/app/apis/kubeadm/validation/validation.go
 	flagSet.BoolVar(
 		skipTokenPrint, options.SkipTokenPrint, *skipTokenPrint,
 		"Skip printing of the default bootstrap token generated by 'kubeadm init'.",
 	)
-	// Note: All flags that are not bound to the cfg object should be whitelisted in cmd/kubeadm/app/apis/kubeadm/validation/validation.go
 	flagSet.BoolVar(
 		dryRun, options.DryRun, *dryRun,
 		"Don't apply any changes; just output what would be done.",
 	)
+	flagSet.BoolVar(
+		uploadCerts, options.UploadCerts, *uploadCerts,
+		"Upload certfificates to kubeadm-certs secret.",
+	)
+	flagSet.BoolVar(
```

Comment:
Does this flag change need to be part of this PR, or broken out separately?

Reply:
Idk, but I can add this on another pr if necessary.

Reply:
nvm, We should remove this flag for now.